tests/tests_suite_test.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017 Red Hat, Inc.
+ *
+ */
+
+package tests_test
+
+import (
+	. "github.com/onsi/ginkgo"
+
+	. "github.com/onsi/gomega"
+
+	"testing"
+
+	"kubevirt.io/qe-tools/pkg/ginkgo-reporters"
+
+	"kubevirt.io/containerized-data-importer/tests"
```

Comment:
Import block needs to be gofmt'd / goimport'd

Reply:
I re-ran goimports and gofmt to double check and nothing changed here.

Reply:
Weird, okay.